11.0.Dockerfile

```diff
@@ -195,6 +182,28 @@ ONBUILD ENV ADMIN_PASSWORD="$ADMIN_PASSWORD" \
             WITHOUT_DEMO="$WITHOUT_DEMO"
 ONBUILD ARG LOCAL_CUSTOM_DIR=./custom
 ONBUILD COPY $LOCAL_CUSTOM_DIR /opt/odoo/custom
+
+# enable setting custom uids for odoo user during build of scaffolds
+ONBUILD ARG UID=1000
+ONBUILD ARG GID=1000
+ONBUILD ENV UID="$UID" \
+            GID="$GID"
+
+# Enable Odoo user and filestore
+ONBUILD RUN groupadd -g $GID odoo \
+    && useradd -md /home/odoo -s /bin/false -u $UID -g $GID odoo \
+    && mkdir -p /var/lib/odoo \
+    && chown -R odoo:odoo /var/lib/odoo \
+    # qa artifacts
```

Comment:
If I'm not wrong, this is illegal syntax.

```suggestion
```

Reply:
the syntax is not illegal (it is a commented out line in a docker file, it does not affect the docker run command definition), i removed the line because the comment was not clear